dev-docs/analytics/conversant.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: analytics
+title: Conversant
+description: Conversant Prebid Analytics Adapter
+modulecode: conversant
+gdpr_supported: true
+usp_supported: true
+coppa_supported: false
+prebid_member: true
+gvl_id: 24
+enable_download: false
```

Comment:
This should be true or it won't show up on the Prebid download page

Reply:
@ChrisHuie thanks good catch, updated